src/js/bindings/publisher.js

```diff
@@ -0,0 +1,23 @@
+// function takes a callback and executes when checkboxes under #hero-filter-publisher is changed.
+export function bindCharacterFilterPublisher(callback) {
+  callback();
+  const allCheckboxes = document.querySelectorAll('#hero-filter-publisher input[type="checkbox"]');
+  // loop over checkboxes w/ event listener to detect if they are checked
+  allCheckboxes.forEach((checkbox) => checkbox.addEventListener('change', () => {
+    callback();
+  }));
+}
+
+// filters from checkboxes
+export function getSelectedFilterPublisher() {
+// selects whatever has hero-filter-publisher id and has checkbox input
+  const allCheckboxes = document.querySelectorAll('#hero-filter-publisher input[type="checkbox"]');
+  // eslint-disable-next-line max-len
+  // from copy of array of items returned above, filters for checked item, returns name of publisher (element.defaultValue)
```

Comment:
How about we either break the comment on line 16 into two lines or shorten it a bit. In my opinion it would be a good idea to do that so we can get rid of `// eslint-disable-next-line max-len ` on line 15.  In a perfect world we only want to disable eslint rules when it is absolutely necessary.